docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -353,6 +353,20 @@ const classes = makeStyles(theme => ({
 
 [This codemod](https://github.com/mui-org/material-ui/tree/HEAD/packages/material-ui-codemod#box-sx-prop) will automatically update your code to the new syntax.
 
+- The `borderRadius` system prop value transformation has been changed. It now expects number and multiplies this value with the `theme.shape.borderRadius` value.
+
+```diff
+-<Box sx={{ borderRadius: 'borderRadius' }}>
++<Box sx={{ borderRadius: 1 }}>
+```
+
+- Renamed `createUnarySpacing` to `createUnaryUnit`. All props of the new function defaults to the values necessary for it to behave identically as `createUnarySpacing`.
+
+```diff
```

Comment:
We do not really need to do this breaking change, we can still export it under the old name too...